script.js

```diff
@@ -1,29 +1,28 @@
+var plays = 0;
+var have_winner = false;
```

Comment:
Who has a winner? The game, right? I mean, _the game has a winner_.

In English, some _Present Tense Verbs_ suffer a slightly variation when used in third person. Some of them gets a final _s_, _es_, _ies_... and some of them is completely changed.

This is the case of the verb _have_: in third person (present tense), it changes to _has_.

I have
You have
He/She/It has

So, rather than `have_winner`, we should have `has_winner`.

Some more links:
http://www.grammar.cl/Present/Verbs_Third_Person.htm
http://www.spanishdict.com/conjugate/has?langFrom=es